ee/api/conversation.py

```diff
@@ -57,6 +58,13 @@ def validate(self, data):
             # NOTE: If content is empty, it means we're resuming streaming or continuing generation with only the contextual_tools potentially different
             # Because we intentionally don't add a HumanMessage, we are NOT updating ui_context here
             data["message"] = None
+        billing_context = data.get("billing_context")
+        if billing_context:
+            try:
+                billing_context = MaxBillingContext.model_validate(billing_context)
+                data["billing_context"] = billing_context
+            except pydantic.ValidationError:
+                raise serializers.ValidationError("Invalid billing context.")
```

Comment:
Nitpick: Would it make sense to extend the logging message a bit more (to explain what's invalid about it), or would it be visible through other cases (like traces)?